spec/20-http_request_header_format.md

```diff
@@ -235,7 +235,7 @@ In order to increase interoperability across multiple protocols and encourage su
 
 Vendors MUST expect the header name in any case (upper, lower, mixed), and SHOULD send the header name in lowercase.
 
-##### tracestate Header Field Values
+### Header Values
```

Comment:
this header will be harder to link to directly with anchor links

Reply:
That would be the same for `Header Name` - it isn't unique either and the document processor is smart enough to create 
* https://www.w3.org/TR/trace-context/#header-name
and
* https://www.w3.org/TR/trace-context/#header-name-0
to distinguish between same headlines.